<ui version="4.0" >
 <class>isoSurfaceDialog</class>
 <widget class="QDialog" name="isoSurfaceDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Iso surfaces</string>
  </property>
  <widget class="QPushButton" name="applyButton" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>410</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Apply</string>
   </property>
   <property name="icon" >
    <iconset/>
   </property>
   <property name="default" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>410</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Ok</string>
   </property>
   <property name="icon" >
    <iconset resource="../ElmerGUI.qrc" >:/icons/dialog-ok-apply.png</iconset>
   </property>
  </widget>
  <widget class="QGroupBox" name="contoursGroup" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title" >
    <string>Contour control</string>
   </property>
   <widget class="QLineEdit" name="contoursMaxEdit" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>90</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>1.0</string>
    </property>
   </widget>
   <widget class="QComboBox" name="contoursCombo" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>30</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="contoursMinEdit" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>90</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="contoursMaxLabel" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>90</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="contoursSpin" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
    <property name="maximum" >
     <number>99999</number>
    </property>
    <property name="value" >
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="variableLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Variable:</string>
    </property>
   </widget>
   <widget class="QLabel" name="contoursLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Contours:</string>
    </property>
   </widget>
   <widget class="QLabel" name="contoursMinLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="keepContourLimits" >
    <property name="geometry" >
     <rect>
      <x>270</x>
      <y>120</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Keep limits</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="colorsGroup" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>170</y>
     <width>391</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title" >
    <string>Color control</string>
   </property>
   <widget class="QLineEdit" name="colorMaxEdit" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>1.0</string>
    </property>
   </widget>
   <widget class="QComboBox" name="colorCombo" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>30</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="colorMinLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QLabel" name="colorLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="colorMinEdit" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="colorMaxLabel" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>60</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="keepColorLimits" >
    <property name="geometry" >
     <rect>
      <x>270</x>
      <y>90</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Keep limits</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="appearanceGroup" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>300</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title" >
    <string>Appearance</string>
   </property>
   <widget class="QCheckBox" name="normalsCheck" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Compute normals</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="featureAngle" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>90</number>
    </property>
    <property name="value" >
     <number>45</number>
    </property>
   </widget>
   <widget class="QLabel" name="featurfeAngleLabel" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Feature angle:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="opacitySpin" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>60</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>100</number>
    </property>
    <property name="value" >
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="opacityLabel" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Opacity:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>410</y>
     <width>77</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
   <property name="icon" >
    <iconset resource="../ElmerGUI.qrc" >:/icons/dialog-close.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../ElmerGUI.qrc" />
 </resources>
 <connections/>
</ui>
